api/src/pcapi/core/fraud/models.py

```diff
@@ -627,3 +627,16 @@ class BlacklistedDomainName(PcObject, Base, Model):
     dateCreated: datetime.datetime = sa.Column(
         sa.DateTime, nullable=False, default=datetime.datetime.utcnow, server_default=sa.func.now()
     )
+
+
+class ProductWhitelist(PcObject, Base, Model):
+    """
+    Add a normally not eligible EAN into a product whitelist
+    """
+
+    title = sa.Column(sa.Text(), nullable=False)
+    ean = sa.Column(sa.String(length=13), nullable=False)
+    dateCreated: datetime.datetime = sa.Column(
+        sa.DateTime, nullable=False, default=datetime.datetime.utcnow, server_default=sa.func.now()
+    )
+    comment = sa.Column(sa.Text)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Quelle différence entre `title = sa.Column(sa.Text(), ...` et `comment = sa.Column(sa.Text)` ?

Ils devraient tous les deux être similaires non ? La ligne `comment = sa.Column(sa.Text)` prend la classe `sa.Text` au lieu d'une instance

Reply:
Aucune idée, c'est copié collé, mais je normalise.